git_machete/client.py

```diff
@@ -414,7 +415,7 @@ def get_root_of(branch: LocalBranchShortName) -> LocalBranchShortName:
                     "cycle in the resulting graph or the candidate is a stale branch"))
             if upstream:
                 debug(
-                    "discover_tree()",
+                    f"{inspect.stack()[0].function}()",
```

Comment:
Hmmm that's pretty ugly... would it be possible to move that logic to `debug` itself? 🤔